packages/worker/src/worker.ts

```diff
@@ -1028,7 +1095,7 @@ export class Worker {
             ifaceName,
             fnName,
           });
-        } else if (dep.callDuringReplay || !info.isReplaying) {
+        } else if (dep.callDuringReplay || !info.unsafe.isReplaying) {
```

Comment:
It's late so I might be missing something but I'm pretty sure we never update `isReplaying`

Reply:
Don't have to fix in this PR, could open an issue but this is pretty critical.

Reply:
We set it from the start workflow activation:

<img width="524" alt="image" src="https://user-images.githubusercontent.com/251288/170601953-4c0efb2a-9946-497e-84ed-2b33eed96b44.png">

And then update it during future activations in `vm.ts` and `worker-interface.ts`:

<img width="1010" alt="image" src="https://user-images.githubusercontent.com/251288/170602147-e34bebec-c79d-4e55-8c49-3fcc3f697b14.png">

<img width="993" alt="image" src="https://user-images.githubusercontent.com/251288/170602123-5ba9d07b-212f-4125-ab46-f34be515a38d.png">

Reply:
Yup which means it doesn't take affect in the main thread where the worker lives and it requires this info to run sink calls.

Reply:
https://github.com/temporalio/sdk-typescript/issues/667